tensorflow_datasets/core/download/checksums.py

```diff
@@ -76,7 +76,7 @@ def _get_sizes_checksums(checksums_path):
   """Returns {URL: (size, checksum)}s stored within file."""
   checksums = {}
   for line in _read_file(checksums_path).split('\n'):
-    if not line:
+    if not line or line.strip().startswith('#'):
```

Comment:
Commits must do a particular job. you have already made this change on #808.

Reply:
Fixed